commands/project/profile/remove.ts

```diff
@@ -0,0 +1,184 @@
+import fs from 'fs';
+import path from 'path';
+import { Argv, ArgumentsCamelCase } from 'yargs';
+import {
+  getAllHsProfiles,
+  getHsProfileFilename,
+  loadHsProfileFile,
+} from '@hubspot/project-parsing-lib';
+import { fetchProject } from '@hubspot/local-dev-lib/api/projects';
+import { deleteProject } from '@hubspot/local-dev-lib/api/projects';
+import { trackCommandUsage } from '../../../lib/usageTracking';
+import { getProjectConfig } from '../../../lib/projects/config';
+import { uiBetaTag } from '../../../lib/ui';
+import { uiLogger } from '../../../lib/ui/logger';
+import { EXIT_CODES } from '../../../lib/enums/exitCodes';
+import { YargsCommandModule, CommonArgs } from '../../../types/Yargs';
+import { makeYargsBuilder } from '../../../lib/yargsUtils';
+import { commands } from '../../../lang/en';
+import { confirmPrompt, listPrompt } from '../../../lib/prompts/promptUtils';
+import { fileExists } from '../../../lib/validation';
+import { debugError } from '../../../lib/errorHandlers';
+
+const command = 'remove [name]';
+const describe = uiBetaTag(commands.project.profile.remove.describe, false);
+
+type ProjectProfileRemoveArgs = CommonArgs & {
+  name?: string;
+};
+
+async function handler(
+  args: ArgumentsCamelCase<ProjectProfileRemoveArgs>
+): Promise<void> {
+  const { derivedAccountId } = args;
+
+  trackCommandUsage('project-profile-remove', undefined, derivedAccountId);
+
+  const { projectConfig, projectDir } = await getProjectConfig();
+
+  if (!projectConfig || !projectDir) {
+    uiLogger.error(commands.project.profile.remove.errors.noProjectConfig);
+    process.exit(EXIT_CODES.ERROR);
+  }
+
+  const projectSourceDir = path.join(projectDir, projectConfig.srcDir);
+  let profileName: string | undefined = undefined;
+
+  if (args.name) {
+    profileName = args.name;
+    const profileFilename = getHsProfileFilename(profileName);
+
+    if (!fileExists(path.join(projectSourceDir, profileFilename))) {
+      uiLogger.error(
+        commands.project.profile.remove.errors.noProfileFound(profileFilename)
+      );
+      process.exit(EXIT_CODES.ERROR);
+    }
+  } else {
+    const existingProfiles = await getAllHsProfiles(projectSourceDir);
+
+    const promptResponse = await listPrompt(
+      commands.project.profile.remove.prompts.removeProfilePrompt,
+      {
+        choices: existingProfiles.map(profile => ({
+          name: getHsProfileFilename(profile),
+          value: profile,
+        })),
+      }
+    );
+
+    if (promptResponse) {
+      profileName = promptResponse;
+    }
+    uiLogger.log('');
+  }
+
+  // This should never happen
+  if (!profileName) {
+    process.exit(EXIT_CODES.ERROR);
+  }
+
+  let targetAccountId: number | undefined;
+
+  try {
+    const profileToRemove = loadHsProfileFile(projectSourceDir, profileName);
```

Comment:
Intentionally not using `loadProfile` here because I don't want any of the logs if it fails to load. I'm loading it because it lets us prompt the user to remove the project from the account if we can find the `accountId` in it.